crates/tiny/src/main.rs

```diff
@@ -142,10 +144,26 @@ fn run(
         for server in servers.iter().cloned() {
             tui.new_server_tab(&server.addr, server.alias);
 
+            let tls = server.tls;
+            let sasl_auth = if let Some(sasl_auth) = &server.sasl_auth {
+                match sasl_from_config(tls, sasl_auth) {
+                    Ok(sasl) => Some(sasl),
+                    Err(e) => {
+                        tui.add_client_err_msg(
+                            &format!("SASL error for server [{}]: {}", server.addr, e),
+                            &MsgTarget::Server { serv: "mentions" },
```

Comment:
Should we show these errors in the server tabs? Line 145 creates the server tab.